src/site/lab6.rst

```diff
@@ -0,0 +1,75 @@
+******
+Lab #6
+******
+
+* Refer to the topic pages for this lab
+    * You will find many of the answers to your questions there
+* Feel free to use your laptop if you have it
+* I strongly encourage you to work with others in the lab
+* Ensure I have recorded your completion. Failure to do so will result in a grade of 0
+* When you get stuck, do me a favour and ask those sitting around you for help
+    * I want people to get used to working together in the labs
+    * Peer teaching and peer learning is super effective
+
+
+Making a Bad Array Queue
+=========================
```

Comment:
I wouldn't call it bad. Such a queue is still a queue, no need to 💩 all over their dreams (especially those who have trouble with this lab).

Maybe, rudimentary?